src/typings/cesium/index.d.ts

```diff
@@ -1 +1,11 @@
-declare var Cesium: any;
+declare const Cesium: any;
+
+declare interface Cartesian2 {
+    x: number,
+    y: number,
+    equals(right): boolean
```

Comment:
there is cartesian3 class under models 
maybe we should delete it and only have typings